DOCS.md

```diff
@@ -11,440 +11,397 @@ Version 1.0.0
 
     <ComponentLibrary id="TruexAdLibrary" uri="http://static.truex.com.s3.amazonaws.com/roku/TruexAdRenderer-Roku-v0.9.0.pkg"/>
 
 ### TruexAdRenderer Input Events
 
-#### init
+#### `init`
 
-    m.tar = m.top.createChild("TruexAdLibrary.TruexAdRenderer")
-    m.tar.observeFieldScoped("event", "handleTarEvent")
-    m.tar.observeFieldScoped("request", "handleTarPlaybackRequest")
+```
+m.tar = m.top.createChild("TruexAdLibrary.TruexAdRenderer")
+m.tar.observeFieldScoped("event", "handleTarEvent")
+m.tar.observeFieldScoped("request", "handleTarPlaybackRequest")
 
-    m.tar.action = {
-        type : "init",
-        creativeURL : "<creative URL returned by SSAI>",
-        adParameters : "<Ad parameters associative array as returned by
-        SSAI>",
-        slotType : "<the type of the current ad pod>"
-    }
+m.tar.action = {
+    type : "init",
+    adParameters : "<Ad parameters associative array as returned by SSAI>",
+    slotType : "<the type of the current ad pod>",
+    supportsUserCancelStream : <optional; set to true to enable the userCancelStream event>
+}
+```
 
-This event will be triggered by the channel code in order to
-instantiate the `TruexAdRenderer`. The renderer will parse out the
-`creativeURL`, `adParameters` and `slotType` passed to it and
-make a request to the TrueX ad server to see what ads are available.
+This event will be triggered by the channel code in order to initialize the `TruexAdRenderer`. The renderer will parse out the `adParameters` and `slotType` passed to it and make a request to the true\[X\] ad server to see what ads are available.
 
-You may instantiate `TruexAdRenderer` early (a few seconds before
-the next pod even starts) in order to give it extra time to make the
-ad request. The renderer will output a `fetchAdComplete` event at
-completion of this ad request. The event can be used to coordinate the
-call to `start` described next.
+You may initialize `TruexAdRenderer` early (a few seconds before the next pod even starts) in order to give it extra time to make the ad request. The renderer will output an `adFetchCompleted` event at completion of this ad request. This event can be used to facilitate the implementation of a timeout or loading indicator, and when to make the call to `start`.
 
 The parameters for this method call are:
 
-* `creativeURL`: TrueX asset url returned by SSAI. In the example of Uplynk, this would correspond to `response.ads.breaks[0].ads[0].creative`
 * `adParameters`: AdParameters as returned by SSAI. In the example of Uplynk, this would correspond to `response.ads.breaks[0].ads[0].adParameters`
 * `slotType`: the type of the current ad pod, `PREROLL` or `MIDROLL`
+* `supportsUserCancelStream`: optional -- set to `true` to enable the [userCancelStream](#usercancelstream) event
+
 
-#### start
+#### `start`
 
-    m.tar.action = {
-        type : "start"
-    }
+```
```

Comment:
Apparently `brightscript` is supported by GitHub markdown. What will they think of next?

**Syntax**:
```
```brightscript
```

**Example**:
```brightscript
m.tar.action = {
    type : "start"
}
```

Reply:
nice, adding